packages/core/src/components/graphs/gauge.ts

```diff
@@ -0,0 +1,356 @@
+			.merge(valueNumber)
+			.style("font-size", `${valueFontSize(radius)}px`)
+			.attr("text-anchor", "middle")
+			.text(d => numberFormatter(d));
+
+		// add the percentage symbol beside the valueNumber
+		const {
+			width: valueNumberWidth
+		} = DOMUtils.getSVGElementSize(DOMUtils.appendOrSelect(svg, "text.gauge-value-number"), { useBBox: true });
+
+		DOMUtils.appendOrSelect(valueNumberGroup, "text.gauge-value-symbol")
+			.style("font-size", `${valueFontSize(radius) / 2}px`)
+			.attr("x", valueNumberWidth / 2)
+			.text("%");
+	}
+
+	/**
+	 * adds the delta number for the gauge
+	 */
+	drawDelta() {
+		const self = this;
+		const svg = this.getContainerSVG();
+		const options = this.model.getOptions();
+		const delta = this.getDelta();
+
+		// Sizing and positions relative to the radius
+		const radius = this.computeRadius();
+		const deltaFontSize = delta ? Tools.getProperty(options, "gauge", "deltaFontSize") : () => 0;
+		const numberFormatter = Tools.getProperty(options, "gauge", "numberFormatter");
+		const arrowSize = Tools.getProperty(options, "gauge", "arrowSize");
+		const numberKerning = Tools.getProperty(options, "gauge", "numberKerning");
+
+		const numbersGroup = DOMUtils.appendOrSelect(svg, "g.gauge-numbers");
+
+		// Add the smaller number of the delta
+		const deltaGroup = DOMUtils.appendOrSelect(
+			numbersGroup,
+			"g.gauge-delta"
+		).attr(
+			"transform",
+			`translate(0, ${deltaFontSize(radius) + numberKerning})`
+		);
+
+		const deltaNumber = deltaGroup.selectAll("text.gauge-delta-number")
+			.data(delta != null ? [delta] : []);
+
+		deltaNumber
+			.enter()
+			.append("text")
+			.merge(deltaNumber)
+			.attr("class", "gauge-delta-number")
+			.attr("text-anchor", "middle")
+			.style("font-size", `${deltaFontSize(radius)}px`)
+			.text(d => `${numberFormatter(d)}%`);
+
+		// Add the caret for the delta number
+		const {
+			width: deltaNumberWidth
+		} = DOMUtils.getSVGElementSize(DOMUtils.appendOrSelect(svg, "g.gauge-delta"), { useBBox: true });
+
+		const deltaArrow = deltaGroup.selectAll("svg.gauge-delta-arrow")
+			.data(delta != null ? [delta] : []);
+
+		deltaArrow
+			.enter()
+			.append("svg")
+			.merge(deltaArrow)
+			.attr("class", "gauge-delta-arrow")
+			.attr("x", -arrowSize(radius) - deltaNumberWidth / 2)
+			.attr("y", -arrowSize(radius) / 2 - deltaFontSize(radius) * 0.35)
+			.attr("width", arrowSize(radius))
+			.attr("height", arrowSize(radius))
+			.attr("viewBox", "0 0 16 16");
+
+		// Needed to correctly size SVG in Firefox
+		DOMUtils.appendOrSelect(deltaArrow, "rect.gauge-delta-arrow-backdrop")
+			.attr("width", "16")
+			.attr("height", "16")
+			.attr("fill", "none");
+
+		// Draw the arrow with status
+		const status = Tools.getProperty(options, "gauge", "status");
+		DOMUtils.appendOrSelect(deltaArrow, "polygon.gauge-delta-arrow-polygon")
+			.classed(`status--${status}`, status != null)
+			.attr("fill", () => status == null ? "currentColor" : null)
```

Comment:
Should it be
```suggestion
			.attr("fill", () => status === null ? "currentColor" : null)
```
?

Reply:
also, wouldn't it render fine anyways without this line?

Reply:
it renders in black 
![image](https://user-images.githubusercontent.com/14351335/84907007-a80dca80-b080-11ea-89df-07179df83b96.png)
rather than the current color which was the intended default i believe.. 
![image](https://user-images.githubusercontent.com/14351335/84907348-13579c80-b081-11ea-91aa-a409b93c178b.png)

we can have it render black and the user can target `.gauge-delta-arrow-polygon`  if you want ??